yt_dlp/extractor/bilibili.py

```diff
@@ -1283,7 +1302,13 @@ def get_metadata(page_data):
 
         def get_entries(page_data):
             for entry in traverse_obj(page_data, ('list', 'vlist')) or []:
```

Comment:
I know it's old code, but let's clean it up while we're here
```suggestion
            for entry in traverse_obj(page_data, ('list', 'vlist', ..., {dict})):
```